python-sdk/docs/astro/sql/operators/load_file.rst

```diff
@@ -288,6 +288,15 @@ Users can also load data from an FTP:
    :start-after: [START load_file_example_21]
    :end-before: [END load_file_example_21]
 
+Loading data from Azure Blob storage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Users can also load data from an FTP:
```

Comment:
this does not look correct.

Reply:
https://github.com/astronomer/astro-sdk/pull/1598/commits/a0954f5da632db36de18e44a9a89ab49b91d3f9c